charts/refarch-templates/README.md

```
@@ -292,4 +326,4 @@ The following configuration is picked up by default:
 ```
 
 > **Note:**: Those defaults are suitable for Spring-based applications, because applications like `refarch-backend` or `refarch-eai` expose those endpoints via Spring Actuator.
-Web components like `refarch-frontend` or `refarch-webcomponent` are compatible as well, as those mimic the actuator API.
+Web components like `refarch-frontend` or `refarch-webcomponent` are compatible as well, as those mimic the actuator API.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)